pyproject.toml

```diff
@@ -1,18 +1,17 @@
 [project]
-name = "ics_cobraOps"
+name = "ics-cobraOps"
 readme = "README.md"
-authors = [
-  {name = "Javier Gracia Carpio", email = "jgracia@mpe.mpg.de"},
-]
+authors = [{ name = "Javier Gracia Carpio", email = "jgracia@mpe.mpg.de" }]
 requires-python = ">=3.9"
 description = "A pacakage that simulates the movements of cobras and calculates their possible collisions"
 license = "GPL-3.0"
 dependencies = [
   "numpy",
   "scipy",
   "matplotlib",
-  "ics_cobraCharmer @ git+https://github.com/Subaru-PFS/ics_cobraCharmer.git@1.4.18",
-  "pfs_instdata @ git+https://github.com/Subaru-PFS/pfs_instdata.git@1.8.44",]
+  "ics-cobraCharmer @ git+https://github.com/Subaru-PFS/ics_cobraCharmer.git",
+  "pfs-instdata @ git+https://github.com/Subaru-PFS/pfs_instdata.git@tickets/INSTRM-2729",
```

Comment:
`pfs-instdata` update was merged so it should now be pip installable from the master branch, so the `tickets/INSTRM-2729` is no longer needed.